test/features/MyFirstApiConsume.feature

```diff
@@ -0,0 +1,36 @@
+Feature: My First Api Consume
+    Scenario: Verify GET Service
+        Given a REST API url
+        When the user consumes a GET Service
+        Then the response should contain a OK status
+        And the response must contain a property "origin"
+
+    Scenario: Verify GET Service with Query
+        Given a REST API url
+        When the user consumes a GET Service with query parameters
+        Then the response should contain a OK status
+        And the response must contain the query values
+
+    Scenario: Verify HEAD Service
+        Given a REST API url
+        When the user consumes a HEAD Service
+        Then the response should contain a OK status
+        And the response body must be empty
+
+    Scenario: Verify PATCH Service with Query
+        Given a REST API url
+        When the user consumes a PATCH Service with query parameters
+        Then the response should contain a OK status
+        And the response must contain the PATCH query name
+
+    Scenario: Verify PUT Service with Query
+        Given a REST API url
+        When the user consumes a PUT Service with query parameters
+        Then the response should contain a OK status
+        And the response must contain the PUT query name
```

Comment:
why the name should contain "PUT"?

I think that your gherkin is very generic and we need to see the implementation in order to understand the test. My suggestion is that the Gherkin should be clear enough. Remember that this file will be read to the client

Reply:
If i don't specify the difference of each reponse, then the test will be ambiguous for Cucumber, if that is the case i will create a query.js in the common file, that contains  "the response must contain the query name" and that will work for all  3 scenarios that do this process